functionalTest/test/specs/example.e2e.js

```diff
@@ -0,0 +1,81 @@
+const dashboard = require('../pageobjects/dashboard.page');
+const reusableObj = require('../utilities/reusableLibrary');
+
+
+describe('Covid-19 Dashboard', () => {
+    it('should have correct title', () => {
+        dashboard.open();
+        console.log("Url Launched....");
+        browser.pause(1000);
+        //  reusableObj.validateTitle('Covid-19 Dashboard');
```

Comment:
Try to avoid commented out, dead code. If you need to comment out something temporarily, say so in a comment, with a date so that people know when it's safe to delete

Reply:
Removed commented code which is not required in the future.